.github/workflows/remove-labels.yml

```diff
@@ -0,0 +1,34 @@
+name: Remove Labels
+
+on:
+  pull_request_branch:
+    types: [labeled]
+
+jobs:
+  label_cleanup:
+    runs-on: ubuntu-latest
+
+    if: |
+      (github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'ci:run')) ||
+      (github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'ci:test'))
+
+    name: remove CI runner labels
+    steps:
+      - name: remove tags
+        uses: actions/github-script@a3e7071a34d7e1f219a8a4de9a5e0a34d1ee1293
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            github.issues.removeLabel({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              name: 'ci:run'
+            })
+            github.issues.removeLabel({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              name: 'ci:test'
+            })
+        continue-on-error: true
```

Comment:
Is it possible to add a comment in the yml for why `continue-on-error: true` is needed?